charts/oes/values.yaml

```diff
@@ -281,6 +281,22 @@ sapor:
       spinAdminUsername: admin
       spinAdminPassword: admin
 
+      ## LDAP
+      ldap:
+        ldapEnabled: true
+        ldapUsername: admin
+        ldapPassword: opsmx@123
+        ldapAdminLoginEnabled: false
+        ldapAdminUsername:
+        ldapAdminPassword:
```

Comment:
We should have some dummy values for these as helm doesn't allow empty value substitution

Reply:
I put this dummy values for username & passwords:
```
  ldapAdminUsername: admin
  ldapAdminPassword: admin
```